src/pages/staticPages/tools/mutationMapper/MutationMapperTool.tsx

```diff
@@ -102,6 +98,7 @@ export default class MutationMapperTool extends React.Component<
                     />
                     {this.store.mutationsNotAnnotated.length > 0 &&
                         this.inputParseWarning()}
+                    {this.shouldShowGrch38Warning && this.grch38Info()}
```

Comment:
We should probably rename either the variable to `shouldShowGrch38Info` or the method to `grch38Warning`